raiden/network/proxies/secret_registry.py

```diff
@@ -61,13 +66,24 @@ def __init__(
         self._open_secret_transactions_lock = Semaphore()
 
     def register_secret(self, secret: Secret, given_block_identifier: BlockSpecification):
-        self.register_secret_batch([secret], given_block_identifier)
+        try:
+            self.register_secret_batch([secret], given_block_identifier)
+        except NoStateForBlockIdentifier:
+            # If the given block identifier is older than 128 blocks then try again
+            # with the latest block. This can happen if we shut down for some time
+            # and we get to register_secret when replaying the state changes.
+            self.register_secret_batch([secret], given_block_identifier)
```

Comment:
How is this using the `latest` block? i think you forgot to change the second param

Reply:
I did